### PR TITLE
Linking to our DietPi Blog page added

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/references.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Links to special software packages are located at the end of the according softw
 
 - Twitter: <https://twitter.com/DietPi_>
 - Facebook: <https://www.facebook.com/groups/167403007229675>
+- DietPi Blog page: <https://dietpi.com/blog>
```
Comment (MichaIng) [inline docs/references.md:10]: What do you think of naming it just "Blog", to have a single word list "Twitter" and "Facebook"?

File: docs/socialmedia.md [hunk 2]
```diff
@@ -6,4 +6,8 @@ DietPi announces are done via Twitter: <https://twitter.com/DietPi_>.
 
 ## DietPi on Facebook
 
-DietPi announces and discussions are done via Facebook: <https://www.facebook.com/groups/167403007229675>
+DietPi announces and discussions are done via Facebook: <https://www.facebook.com/groups/167403007229675>.
```
Comment (MichaIng) [inline docs/socialmedia.md:9]: I think after the colon, a dot is not required anymore to end the sentence, is it?

File: docs/references.md (hunk 1 above)
Comment (StephanStS) [reply docs/references.md:10]: Ok.

File: docs/socialmedia.md (hunk 2 above)
Comment (StephanStS) [reply docs/socialmedia.md:9]: Ok. Changed for all entries.

**Pull request merged**